src/hal/drivers/mesa-hostmot2/sserial.c

```diff
@@ -1344,14 +1344,14 @@ void hm2_sserial_prepare_tram_write(hostmot2_t *hm2, long period){
                                         double temp = *pin->float_pin;
                                         memcpy(&buff, &temp, sizeof(double));
                                     } else {
-                                        HM2_ERR_NO_LL("sserial write: LBP_FLOAT of bit-length %i not handled\n", conf->DataLength);
-                                        conf->DataType = 0; // only warn once, then ignore
+                                        HM2_ERR_NO_LL("sserial read: LBP_FLOAT of bit-length %i not handled\n", conf->DataLength);
```

Comment:
Why change this from "write" to read"?  It's in the hm2_sserial_prepare_tram_write() function, i think the original message is correct.